addons/website/static/src/js/content/website_root.js

```diff
@@ -33,7 +34,9 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         KeyboardNavigationMixin.init.call(this, {
             autoAccessKeys: false,
         });
-        return this._super(...arguments);
+        this._super(...arguments);
+        this.fullscreenIndication = new FullscreenIndication(this);
+        this.fullscreenIndication.appendTo(document.body);
```

Comment:
Because `Widget.appendTo` is asynchronous, you need to make sure it is resolved before you exit the method, which you can't do inside init (because it's synchronous), create a new override for the `willStart` method and do the creating and appending inside that.